applications/mne_analyze/plugins/averaging/averaging.pro

```diff
@@ -130,3 +131,20 @@ contains(MNECPP_CONFIG, useFFTW):!contains(MNECPP_CONFIG, static) {
                 -lfftw3_threads \
     }
 }
+
+################################################## BUILD TIMESTAMP/HASH UPDATER ############################################
```

Comment:
Can we put the code below into a function inside the mne-cpp.pri file? Would save a lo of redundant code.

Reply:
not sure, there are some variables that are context dependent. but if you know and want to try, go ahead. just give me a heads up before pushing.